paasta_tools/cli/cmds/validate.py

```diff
@@ -692,70 +712,100 @@ def validate_autoscaling_configs(service_path: str) -> bool:
                 should_skip_cpu_override_validation = (
                     service in skip_cpu_override_validation_list
                 )
+                seen_provider_types: Set[str] = set()
+                configured_provider_count = len(autoscaling_params["metrics_providers"])
 
-                try:
-                    _validate_autoscaling_config(autoscaling_params)
-                    if autoscaling_params["metrics_provider"] == "active-requests":
-                        _validate_active_requests_autoscaling_configs(
-                            instance_config, autoscaling_params
-                        )
+                for metrics_provider in autoscaling_params["metrics_providers"]:
+                    try:
+                        # Generic validation of the config
+                        _validate_autoscaling_config(metrics_provider)
 
-                    elif autoscaling_params["metrics_provider"] == "arbitrary_promql":
-                        _validate_arbitrary_promql_autoscaling_configs(
-                            autoscaling_params
-                        )
+                        # Multi-metrics specific validation:
+                        # 1. Bespoke policies shouldn't also use CPU
```

Comment:
i think it'd be worth documenting this with something a little more strongly worded lest someone "clean" this up by only checking for cpu autoscaling in conjunction with bespoke autoscaling

```suggestion
                        # 1. Bespoke policies cannot use multi-metric scaling
```

Reply:
Whoops, yea, definitely brain-fart